packages/atomic/src/components/atomic-pager/atomic-pager.tsx

```diff
@@ -52,6 +52,11 @@ export class AtomicPager implements InitializableComponent {
   };
   @State() error!: Error;
 
+  @Event({
```

Comment:
Would there be other components that needs to trigger this ? 🤔  

I don't think so... Maybe results per page ?

Reply:
Results per page could if you lessen the number of results per page.
Facet values could too, like in the JSUI (perhaps an as option)
The point of the event is too allow anything inside the search interface to trigger a scroll top

Reply:
So, do we need to make it generally available to all components ? Like, the "name" of the event (`atomic/scrollToTop`)

Reply:
It is available, the thing is when using decorator, you can't put a variable inside of it, only primitive values!